### PR TITLE
Chapter 3 - Resilience

### PR DESCRIPTION
**Pull request by darienmt**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -50,6 +50,7 @@
   * [Azure Cloud Storage](./content/chapter-03/AzureCloudStorage.md)
   * [Oracle Cloud Storage](./content/chapter-03/OracleCloudStorage.md)
 * [Intermittent connectivity issues](./content/chapter-03/IntermittentConnectivity.md)
+* [Resilience of software design](./content/chapter-03/Resilience.md)
```
Comment (vkhazin) [inline SUMMARY.md]: Resilience of software design => Resilient Software Design

File: content/chapter-03/Intro.md [hunk 2]
```diff
@@ -10,6 +10,5 @@
 * Lab: Instance types and and IO considerations
 * [Intermittent connectivity issues](./IntermittentConnectivity.md)
 * Lab: What to do when I cannot connect?
-* Caching and Session Management
-* Resilience of software design
+* [Resilience of software design](Resilience.md)
```
Comment (vkhazin) [inline content/chapter-03/Intro.md]: Resilience of software design => Resilient Software Design

File: content/chapter-03/Resilience.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Resilience of software design
```
Comment (vkhazin) [inline content/chapter-03/Resilience.md]: Resilience of software design => Resilient Software Design

**Review by vkhazin — CHANGES_REQUESTED** [review]

File: SUMMARY.md (hunk 1 above)
Comment (darienmt) [reply SUMMARY.md]: Done

File: content/chapter-03/Intro.md (hunk 2 above)
Comment (darienmt) [reply content/chapter-03/Intro.md]: Done

File: content/chapter-03/Resilience.md (hunk 3 above)
Comment (darienmt) [reply content/chapter-03/Resilience.md]: Done

**Pull request merged**